Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby ">= 2.4"
+# ruby ">= 2.4"
```

Comment:
Ups. This needs fixing...

Reply:
The problem with this is because the `ruby2.4-rubygems-bundler` package contains the bundler 1.10.6. This version is from July 2015. This constraint only works with bundler >= 1.12.x.

Reply:
I wonder if we can remove this line altogether. It has only caused headaches for us and we already know which versions are supposed to work with ruby 2.4 😅